src/main/java/com/exasol/bucketfs/UploadResult.java

```diff
@@ -0,0 +1,26 @@
+package com.exasol.bucketfs;
+
+/**
+ * This class contains the result of an upload operation.
+ */
+public class UploadResult {
+    private final boolean wasUploadNecessary;
+
+    /**
+     * Create a new instance of {@link UploadResult}.
+     * 
+     * @param wasUploadNecessary if the upload was necessary
+     */
+    UploadResult(final boolean wasUploadNecessary) {
```

Comment:
```suggestion
    public UploadResult(final boolean wasUploadNecessary) {
```

Reply:
why should the conmstructor be public. It's only built by the object others only need to read this object